apps/dashboard/app/apps/router.rb

```diff
@@ -1,3 +1,5 @@
+# Generic Router class (as a pose to SysRouter that is specific to system apps)
```

Comment:
```suggestion
# Generic Router class (as apposed to SysRouter that is specific to system apps)
```
Not sure if this is maybe what you meant?

Reply:
LOL yes.